content/reference/endpoints/fees.md

```diff
@@ -44,4 +62,36 @@ Returns a single fee identified by ID.
 GET /fees/:fee_id
 ~~~
 
-<%= render 'json_response', endpoint: "fees", scopes: %w(public) %>
+<%= render 'json_response', endpoint: "fees", scopes: [{ public: "public-rates_write" }] %>
+
+## Create a new fee
+
+Returns a newly created fee.
+
+~~~
+POST /fees
+~~~
+
+<%= render 'json_response', endpoint: "fees", request: "create",
+  scopes: [{ rates_write: "public-rates_write" }] %>
+
+## Update a fee
+
+Returns an updated fee identified by ID.
+
+~~~
+PUT /fees/:fee_id
+~~~
+
+<%= render 'json_response', endpoint: "fees", request: "update",
+  scopes: [{ rates_write: "public-rates_write" }] %>
+
+## Destroy a fee
+
+Required OAuth scope: `:rates_write`
```

Comment:
I think good to explain that it's really getting archived and looking at `archived_at` is the way to identify if a `fee` should not longer be considered as valid.